11-protected-routes/src/App.js

```diff
@@ -0,0 +1,37 @@
+import { BrowserRouter as Router, Routes, Route } from 'react-router-dom';
```

Comment:
Awesome job everyone! Thank you for taking the React class with Geekwise Academy!